### PR TITLE
311 Petition Clerk Start A Case Button

### PR DESCRIPTION
**Pull request by wbyoko**
Screenshot 2019-03-25 12 45 26

Screenshot 2019-03-25 12 45 34

### CODE REVIEW

File: shared/src/business/entities/Case.js [hunk 1]
```diff
@@ -669,7 +669,7 @@ Case.getProcedureTypes = () => {
  * @returns {string[]}
  */
 Case.getFilingTypes = userRole => {
-  return FILING_TYPES[userRole];
+  return FILING_TYPES[userRole] || FILING_TYPES.petitioner;
```
Comment (kkoskelin) [inline shared/src/business/entities/Case.js:672]: Did you mean `return FILING_TYPES[userRole || FILING_TYPES.petitioner];` ?

**Review by kkoskelin — DISMISSED** [review]
I think a change is needed, perhaps with a test to cover it.

File: shared/src/business/entities/Case.js (hunk 1 above)
Comment (wbyoko) [reply shared/src/business/entities/Case.js:672]: This defaults petitioner filing types as default. There were tests added for this. What you are suggesting passes an array as the property.

Comment (kkoskelin) [reply shared/src/business/entities/Case.js:672]: I continue to have a case of the Mondays.  Withdrawn!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**